src/main/java/oyster/tasks/TaskList.java

```diff
@@ -0,0 +1,110 @@
+package oyster.tasks;
+
+import java.util.ArrayList;
+
+/**
+ * TaskList class that holds Tasks.
+ */
+public class TaskList {
+    private final ArrayList<Task> tasks;
+
+    public TaskList() {
+        tasks = new ArrayList<Task>();
+    }
+
+    /**
+     * Insert a Task into the list.
+     *
+     * @param task Inserts a Task at the end.
+     */
+    public void insert(Task task) {
+        tasks.add(task);
+    }
+
+    /**
+     * Delete a Task in the list.
```

Comment:
```suggestion
     * Deletes a Task in the list.
```
Do make the same change for the subsequent comments in this file.